### PR TITLE
appium 1.5 cleanup tasks

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

**Comment by Jonahss** [comment]
@imurchie for review

**Comment by imurchie** [comment]
You can remove `helpers. truncateForPrinting`, replacing with lodash's [trunc](https://lodash.com/docs#trunc).

File: lib/webkit-remote-debugger.js [hunk 1]
```diff
@@ -36,17 +36,17 @@ export default class WebKitRemoteDebugger extends RemoteDebugger {
     let pageElementJSON = await this.getJsonFromUrl(this.host, this.port, '/json');
     log.debug(`Page element JSON: ${JSON.stringify(pageElementJSON)}`);
     // Add elements to an array
-    let newPageArray = [];
-    for (let pageObject of pageElementJSON) {
+    let newPageArray = pageElementJSON.map((pageObject) => {
```
Comment (imurchie) [inline lib/webkit-remote-debugger.js]: Nice!

**Comment by Jonahss** [comment]
That's a nice lodash command

**Pull request merged**